build/build.css

```diff
@@ -753,6 +753,34 @@ hr:nth-of-type(6) {
   background-image: url("metalsmith.io/images/divider-6.png");
 }
 
+hr:nth-of-type(7) {
+  background-image: url("metalsmith.io/images/divider-1.png");
+}
+
```

Comment:
Interesting. I didn't even realize `build/` was checked in to git. @lambtron, how are builds normally handled for this site?